crates/biome_rule_options/src/use_iterable_callback_return.rs

```diff
@@ -3,4 +3,18 @@ use serde::{Deserialize, Serialize};
 #[derive(Default, Clone, Debug, Deserialize, Deserializable, Merge, Eq, PartialEq, Serialize)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase", deny_unknown_fields, default)]
-pub struct UseIterableCallbackReturnOptions {}
+pub struct UseIterableCallbackReturnOptions {
+    /// When set to `false`, rule will skip reporting `forEach` callbacks that return a value.
+    #[serde(skip_serializing_if = "Option::<_>::is_none")]
+    pub check_for_each: Option<bool>,
+}
+
+impl UseIterableCallbackReturnOptions {
+    pub const DEFAULT_CHECK_FOR_EACH: bool = true;
```

Comment:
The default for the source rule is `false`. Matching the source rule is important because it makes migrating from eslint easier for users.

 If we keep the default as `true`, IMO we can ship this in a patch release and then switch it to default to match the source rule later in a minor release. Or we could switch the default to `false` and ship this in a minor.